### PR TITLE
4.13.24 upgrade notification 

### PR DESCRIPTION
**Pull request by tbaker1313**

### CODE REVIEW

**Review by wmhutchison — APPROVED** [review]
👍

File: notifications/2024-01-07_upgrades.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+CHG0055022 - MCS GOLDDR - 2024 Q1 Patching - Hard Prune of Image registry, ETCD Defrag and F5 Upgrade
+
+CHG0055020 - MCS GOLDDR - 2024 Q1 Patching - Upgrade to OCP 4.13.24
+
+These six changes represent the work involved to upgrade the production clusters from Openshift 4.12 to 4.13.
+
+**When?**
+
+CHG0055023 - MCS SILVER - 2024 Q1 Patching - Hard Prune of Image registry, ETCD Defrag and F5 Upgrade will commence at 0600 on Sunday January 7 and run as late as 1000.
```
Comment (StevenBarre) [inline notifications/2024-01-07_upgrades.md:19]:
```suggestion
CHG0055023 - MCS SILVER - 2024 Q1 Patching - Hard Prune of Image registry, ETCD Defrag and F5 Upgrade will commence at 6:00AM on Sunday January 7 and run as late as 10:00AM.
```

Comment (tbaker1313) [reply notifications/2024-01-07_upgrades.md]: Fixed in all the 3 sections. I am so used to using military time but agree that either we use the AM/PM style or the military style, so will go with AM/PM type.

Comment (StevenBarre) [reply notifications/2024-01-07_upgrades.md]: I think the customers find AM/PM clearer https://chat.developer.gov.bc.ca/group/internal-platform-services?msg=ubx7Jw3XNp5TjQmnY

**Review by StevenBarre — APPROVED** [review]

**Pull request merged**